index.md

```diff
@@ -54,8 +54,10 @@ for each level and a very brief introduction to plotting.
 
 ## Data
 
-Data files for the lesson are available here: ({{page.dataurl %}})[{{page.dataurl %}}]
+Data files for the lesson are available here: 
+[{{page.dataurl %}}]({{page.dataurl %}})
 
+Please download the **.csv* (Comma Separated Value) files.
```

Comment:
I thought about listing them explicitly and then got lazy :-) I'll do that.
About the missing * it's actually not it's supposed to be '*.csv' in italics in MD (so the forst star is formatting, second is wildcard and third is closing the formatting.